.github/workflows/package_memgraph.yaml

```diff
@@ -28,268 +23,392 @@ on:
+      push_to_s3: ${{ github.event.inputs.push_to_s3 }}
+      s3_bucket: ${{ github.event.inputs.s3_bucket }}
+      s3_region: ${{ github.event.inputs.s3_region }}
+      s3_dest_dir: ${{ github.event.inputs.s3_dest_dir }}
+    secrets: inherit
+
+  Debian12-arm:
+    if: ${{ (github.event.inputs.target_os == 'debian-12-arm' || github.event.inputs.target_os == 'all') && github.event.inputs.toolchain == 'v5' }}
+    uses: ./.github/workflows/reusable_package.yaml
+    with:
+      os: "debian-12-arm"
+      toolchain: ${{ github.event.inputs.toolchain }}
+      arch: "arm"
+      runner_arch_label: "ARM64"
+      build_type: ${{ github.event.inputs.build_type }}
+      timeout_minutes: 120
+      artifact_name: "debian-12-aarch64"
+      push_to_s3: ${{ github.event.inputs.push_to_s3 }}
+      s3_bucket: ${{ github.event.inputs.s3_bucket }}
+      s3_region: ${{ github.event.inputs.s3_region }}
+      s3_dest_dir: ${{ github.event.inputs.s3_dest_dir }}
+    secrets: inherit
+
+  Docker-v4:
+    if: ${{ (github.event.inputs.target_os == 'docker' || github.event.inputs.target_os == 'all') && github.event.inputs.toolchain == 'v4' }}
+    uses: ./.github/workflows/reusable_docker_package.yaml
+    with:
+      os: "debian-11"
+      toolchain: ${{ github.event.inputs.toolchain }}
+      arch: "amd"
+      runner_arch_label: "X64"
+      build_type: ${{ github.event.inputs.build_type }}
+      timeout_minutes: 60
+      additional_build_args: "--for-docker"
+      artifact_name: "docker"
+      push_to_s3: ${{ github.event.inputs.push_to_s3 }}
+      s3_bucket: ${{ github.event.inputs.s3_bucket }}
+      s3_region: ${{ github.event.inputs.s3_region }}
+      s3_dest_dir: ${{ github.event.inputs.s3_dest_dir }}
+    secrets: inherit
+
+  Docker-v5:
+    if: ${{ (github.event.inputs.target_os == 'docker' || github.event.inputs.target_os == 'all') && github.event.inputs.toolchain == 'v5' }}
+    uses: ./.github/workflows/reusable_docker_package.yaml
+    with:
+      os: "debian-12"
+      toolchain: ${{ github.event.inputs.toolchain }}
+      arch: "amd"
+      runner_arch_label: "X64"
+      build_type: ${{ github.event.inputs.build_type }}
+      timeout_minutes: 60
+      additional_build_args: "--for-docker"
+      artifact_name: "docker"
+      push_to_s3: ${{ github.event.inputs.push_to_s3 }}
+      s3_bucket: ${{ github.event.inputs.s3_bucket }}
+      s3_region: ${{ github.event.inputs.s3_region }}
+      s3_dest_dir: ${{ github.event.inputs.s3_dest_dir }}
+    secrets: inherit
+
+  Docker-v4-arm:
+    if: ${{ (github.event.inputs.target_os == 'docker-arm' || github.event.inputs.target_os == 'all') && github.event.inputs.toolchain == 'v4' }}
+    uses: ./.github/workflows/reusable_docker_package.yaml
+    with:
+      os: "debian-11-arm"
+      toolchain: ${{ github.event.inputs.toolchain }}
+      arch: "arm"
+      runner_arch_label: "ARM64"
+      build_type: ${{ github.event.inputs.build_type }}
+      timeout_minutes: 120
```

Comment:
Is it possible to derive the `timeout_minutes` from the `arch` value? Ideally in some global manner so it would still hold true when we add a new hypothetical MemgraphLinux which could run on both x64 or arm.